### PR TITLE
adding 'resolveall' method to api

### PR DESCRIPTION
**Pull request by jackgrossman18**
ref #1456

### CODE REVIEW

**Comment by jackgrossman18** [comment]
cc @maxgrossman

File: modules/Hoot/managers/api.js [hunk 1]
```diff
@@ -574,6 +574,31 @@ export default class API {
...
+        const params = {
+            path: '/job/review/resolveall',
+            method: 'PUT',
+            data: {mapId: mapId}
+        };
+
+        return this.request( params )
+        .then( resp => {
+            if (resp.status !== 200) {
+                return Promise.reject({
```
Comment (maxgrossman) [inline modules/Hoot/managers/api.js]: I think you can just do the alert stuff, no need for the internal promise.reject

File: modules/Hoot/ui/conflicts/resolve.js [hunk 2]
```diff
@@ -85,7 +90,7 @@ export default class Resolve {
                 color: layer.color
             };
 
-        // enter controller refresh state
+        // enter controller refresh stats
```
Comment (maxgrossman) [inline modules/Hoot/ui/conflicts/resolve.js]: I think this was intended to say state.

**Comment by jackgrossman18** [comment]
@maxgrossman made the changes you requested

**Pull request merged**